client/src/fabric/FabricRegistry.ts

```diff
@@ -69,7 +69,8 @@ export abstract class FabricRegistry<T extends FabricRegistryEntry> {
         if (!this.exists(name)) {
             throw new Error(`Entry "${name}" in Fabric registry "${this.registryName}" does not exist`);
         }
-        const oldEntries: T[] = this.getEntries();
+        const oldEntries: T[] = this.getEntries().filter((entry: T) => !entry['managedRuntime']); // Get all entries, but filter out managed runtimes
```

Comment:
FabricRegistry is supposed to be quite abstract but this is adding in implementation specific stuff